sharepoint/sharepoint-ps/sharepoint-server/Move-SPSite.md

```diff
@@ -34,7 +34,7 @@ A no-access lock is applied to the site collection to prevent users from alterin
 
 Once the move is complete, the site collection is returned to its original lock state. 
 
-An IIS reset is required for the site move to take effect.
+An IIS reset is required on all Front-End servers for the site move to take effect. This is not required for SharePoint Server 2019.
```

Comment:
I would say 'any server running the Microsoft SharePoint Foundation Web Application service'.